vnext/PropertySheets/Warnings.props

```diff
@@ -10,7 +10,7 @@
         C4458 - declaration of 'identifier' hides class member
         C4702 - unreachable code
       -->
-    <OfficePreDisabledWarnings>4456;4702;4458</OfficePreDisabledWarnings>
+    <OfficePreDisabledWarnings>4456;4702</OfficePreDisabledWarnings>
```

Comment:
Why are we removing this warning? I think this may have come from your previous PR.

Reply:
Yes I think so - I'll revert it